plutus-ledger/src/Ledger/Orphans.hs

```diff
@@ -45,14 +45,10 @@ instance OpenApi.ToSchema Crypto.XPub where
     declareNamedSchema _ = pure $ OpenApi.NamedSchema (Just "PubKey") mempty
 instance OpenApi.ToSchema Crypto.XPrv where
     declareNamedSchema _ = pure $ OpenApi.NamedSchema (Just "PrvKey") mempty
-
-deriving instance {-# INCOHERENT #-} OpenApi.ToSchema (LogMessage JSON.Value)
-
-deriving instance {-# INCOHERENT #-} OpenApi.ToSchema LogLevel
-
+deriving instance OpenApi.ToSchema (LogMessage JSON.Value)
+deriving instance OpenApi.ToSchema LogLevel
 instance {-# INCOHERENT #-} OpenApi.ToSchema JSON.Value where
```

Comment:
Sorry @kelizarov - Do you mind removing this last `INCOHERENT` here; then it will be good to merge! :)

Reply:
Missed that. Thanks for pointing out :)